cinema/serializers.py

```diff
@@ -132,6 +149,30 @@
                 Ticket.objects.create(order=order, **ticket_data)
             return order
 
+    def update(self, instance, validated_data):
+        tickets_data = validated_data.get("tickets", [])
```

Comment:
The `get` method is used to retrieve `tickets` from `validated_data`, which defaults to an empty list if not present. Ensure that this behavior aligns with your application's logic, as it might lead to unexpected results if `tickets` is a required field.